cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -403,13 +414,145 @@ include::../indexes/drop-an-index.asciidoc[leveloffset=+1]
 include::../indexes/drop-a-non-existing-index.asciidoc[leveloffset=+1]
 
 
+[[indexes-future-indexes]]
+== Future indexes
+
+There are preparations for introducing two new types of indexes, point and range indexes.
+They cannot be used in queries yet, only created and dropped for migration purposes.
+These new index types together with text indexes will replace the current b-tree indexes.
+For more details on these new types, see the operations manual.
+// TODO: add the link to the relevant operations manual section when that section exists
+
+Like b-tree indexes, range indexes are created on one or more properties for all nodes or relationships with a given label or relationship type:
+
+* An index that is created on a single property for any given label or relationship type is called a _single-property index_.
```

Comment:
```suggestion
* An index created on a single property for any given label or relationship type is called a _single-property index_.
```

Reply:
this would be in 2 places then since I just copied this down here